.github/workflows/ci.yml

```diff
@@ -59,6 +59,7 @@ jobs:
         try-scenario:
           - ember-lts-3.28
           - ember-lts-4.12
```

Comment:
@jelhan shouldn't we add the whole matrix of LTS versions in v4 and v5, like 4.4, 4.8, 5.4, 5.8?

Reply:
Is there a realistic risk that it works for 4.12 and 5.12 but not for interim releases? CI minutes are free for open source projects on GitHub. But we should nevertheless review the value for each scenario as they have other trade-offs such as environmental impact.

Happy to add if you feel they provide real value.

Reply:
@jelhan very good question and considering ember-source had very little changes during v4 and v5, I guess answer is "no". It's more like a convention I've seen around in the ecosystem and been following for the addons I've tried to maintain.